components/Tabs/index.js

```diff
@@ -1,9 +1,34 @@
 // Step 2: Create Tabs
 // -----------------------
-// Using axios send a GET request to the address: https://lambda-times-backend.herokuapp.com/topics
+// Using axios send a GET request to the address: 
+// https://lambda-times-backend.herokuapp.com/topics
 // Once the data is returned console.log it and review the structure.
-// Iterate over the topics creating a new Tab component and add it to the DOM
-// under the .topics element.
-//
+const topicsDiv = document.querySelector('.topics');
+
+axios.get('https://lambda-times-backend.herokuapp.com/topics')
+    .then(response => {
+        //console.log(response);
+        //console.log(response.data);
+
+        // Iterate over the topics creating a new Tab component and add it to the DOM
+        // under the .topics element.
+        response.data.topics.forEach(topic => {
+            //console.log(topic);
+            topicsDiv.appendChild(createTab(topic));
+        });
+    })
+    .catch(err => {
+        console.log("Error message: ", err);
```

Comment:
I could be incorrect. But I believe with the .catch() the param we're passing automatically will start with "Error message" Just a fun fact.